package.json

```diff
@@ -13,5 +15,46 @@
   "bugs": {
     "url": "https://github.com/atomixinteractions/material-theme-generator/issues"
   },
-  "homepage": "https://github.com/atomixinteractions/material-theme-generator#readme"
+  "homepage": "https://github.com/atomixinteractions/material-theme-generator#readme",
+  "devDependencies": {
+    "babel-core": "^6.21.0",
```

Comment:
the last version is 6.22.1. Suggest update all packages to latest versions. You can use "npm-check" package.